pkg/storage/utils/decomposedfs/tree/tree.go

```diff
@@ -265,7 +270,27 @@ func (t *Tree) CreateDir(ctx context.Context, n *node.Node) (err error) {
 	// make child appear in listings
 	err = os.Symlink("../"+n.ID, filepath.Join(t.lookup.InternalPath(n.ParentID), n.Name))
 	if err != nil {
-		return
+		// no better way to check unfortunately
+		if !strings.Contains(err.Error(), "file exists") {
+			return
+		}
+
+		// try to remove the node
+		e := t.Delete(ctx, n)
+		switch {
+		case e != nil:
+			appctx.GetLogger(ctx).Debug().Err(e).Msg("cannot move to trashcan")
```

Comment:
rename: trashcan -> trashbin

Reply:
Changed in https://github.com/cs3org/reva/pull/2383